src/mudlet.cpp

```diff
@@ -1913,7 +1915,6 @@ void mudlet::show_options_dialog()
     connect(dactionReconnect, SIGNAL(triggered()), pDlg->need_reconnect_for_data_protocol, SLOT(hide()));
     connect(actionReconnect, SIGNAL(triggered()), pDlg->need_reconnect_for_specialoption, SLOT(hide()));
     connect(dactionReconnect, SIGNAL(triggered()), pDlg->need_reconnect_for_specialoption, SLOT(hide()));
-    if( ! pDlg ) return;
```

Comment:
Actually we need to make a `dlgProfilePreferences * mudlet::mpOptionsDialog` member and only allow this code to create a new instance if there is not one currently in play as part of the fixups to address #852...!